owasp_zap_historic/app.py

```diff
@@ -323,15 +323,15 @@ def use_db(db_name, use_dict=False):
     return cursor
 
 
-def sort_tests(data_list):
-    """Test sorting method."""
-    out = {}
-    for elem in data_list:
-        try:
-            out[elem[1]].extend(elem[2:])
-        except KeyError:
-            out[elem[1]] = list(elem)
-    return [tuple(values) for values in out.values()]
+# def sort_tests(data_list):
+#     """Test sorting method."""
+#     out = {}
+#     for elem in data_list:
+#         try:
+#             out[elem[1]].extend(elem[2:])
+#         except KeyError:
+#             out[elem[1]] = list(elem)
+#     return [tuple(values) for values in out.values()]
```

Comment:
Whats this?

Reply:
That is a function that I am 99% sure isn't being used. I was going to wait to remove it completely after I have worked on the docker unit testing and created another release. Should I just remove it now and revert back to a previous commit if there are issues?

Reply:
You can just leave it commented till you see fit, just checking